packages/admin/src/Support/RelationManagers/BaseRelationManager.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Lunar\Admin\Support\RelationManagers;
+
+use Filament\Resources\RelationManagers\RelationManager;
+use Livewire\Attributes\On;
+
+class BaseRelationManager extends RelationManager
+{
+    #[On('refresh-relation-manager')]
+    public function refreshRelationManager()
+    {
+        // force a re-render
+    }
+}
```

Comment:
```suggestion
#[On('refresh-relation-manager')]
class BaseRelationManager extends RelationManager
{
    //
}
```

@glennjacobs this should work

Reply:
@ryanmitchell does that work for you?

Reply:
I've updated it - I've no way of testing at the moment so I'm taking @wychoong's word for it!